app/views/figures/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= render 'banner' %>
```

Comment:
```suggestion
<%= render 'banner', header: 'Welcome to Ara-Ara', text: 'The best place to find and share your favorite figures' %>
```

Reply:
i don't know if this will work. but hopefully something liek that so that the banner is reusable in other pages